src/client/features/interactions/index.js

```diff
@@ -72,7 +73,8 @@ class Interactions extends React.Component {
     let appBar = h('div.app-bar.interactions-bar', [
       h('div.app-bar-branding', [
         h(PcLogoLink),
-        h('div.app-bar-title', sources.join(', ') + ' Interactions')
+        sources.length === 1 ?  h('div.app-bar-title', `Top ${config.MAX_SIF_NODES} interactions involving ${sources[0]}`):
```

Comment:
should we always display max sif nodes config value if there are less nodes than that?  e.g. top 25 interactions between mdm2 and tp53 when there are only two actual nodes?